packages/kbn-securitysolution-constants/src/index.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * Value list routes
+ */
+export const LIST_URL = '/api/lists';
+export const LIST_INDEX = `${LIST_URL}/index`;
+export const LIST_ITEM_URL = `${LIST_URL}/items`;
+export const LIST_PRIVILEGES_URL = `${LIST_URL}/privileges`;
```

Comment:
I would maybe just add:
```suggestion
export const LIST_INDEX = '.lists';
export const LIST_ITEM_INDEX = '.items';
```

I'd mentioned this effort over in this [PR](https://github.com/elastic/kibana/pull/98125/files#diff-157daa365722ba58f94ba765e86480456ec7a39409291d27c29cbb1a95e2509dR31).

Reply:
Didn't mean for it to show deleting line 15, just an addition...

Reply:
```ts
export const LIST_INDEX = '.lists';
```

Isn't that for the mocks? There's two constants with the same name where one is for the API is listed as this already:

```ts
export const LIST_INDEX = `${LIST_URL}/index`;
```

vs the other which is for the mocks as far as I can.

Reply:
The existing one is for the mocks, the one linked in that PR is being added to security solution constants I believe.

Reply:
Chatted offline. Adding more as we go instead of anticipating needs is best so I retract my comment :)